File: DroidsAPI/src/main/java/com/github/rochedo098/droidsapi/api/types/MemberType.java
package com.github.rochedo098.droidsapi.api.types;

public enum MemberType {
    HEAD("head", "organic"),
    LEFT_ARM("left_arm", "organic"),
    RIGHT_ARM("right_arm", "organic"),
    LEFT_LEG("left_leg", "organic"),
    RIGHT_LEG("right_leg", "organic");

    public String status;
    private final String id;

    MemberType(String id, String status) {
        this.id = id;
        this.status = status;
    }

    public String toString() {
        return this.id;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public String getStatus() {
        return this.status;
    }
}

File: src/main/kotlin/com/github/rochedo098/droids/api/types/MemberType.kt
package com.github.rochedo098.droids.api.types

enum class MemberType(private val id: String, var status: String) {
    HEAD("head", "organic"), LEFT_ARM("left_arm", "organic"), RIGHT_ARM("right_arm", "organic"), LEFT_LEG(
        "left_leg",
        "organic"
    ),
    RIGHT_LEG("right_leg", "organic");

    override fun toString(): String {
        return id
    }
}